Changes.md

```diff
@@ -1,7 +1,13 @@
 1.4.x.x (relative to 1.4.0.0b1)
 =======
 
+Breaking Changes
+----------------
 
+- CyclesOptions :
+  - Removed `useFrameAsSeed` plug. The frame is now automatically used as the seed if `seed` is not set.
+  - Removed `cryptomatteAccurate`. This feature is no longer present in Cycles.
+  - Removed all texture cache options. These had never been exposed in the UI because this never became an offical Cycles feature.>>>>>>> bb7bdf0cb (CyclesOptions, CyclesRenderer : Remove `useFrameAsSeed`)
```

Comment:
A shame they never got that in, I was using this one with special Blackbird builds but a good call to just remove it until they merge something in later...